docs/copilot/chat/copilot-chat.md

```diff
@@ -146,16 +146,15 @@ Chat supports vision capabilities, which means you can attach an image as contex
 
 ## Chat history
 
-As you iterate and send multiple chat prompts in a chat session, VS Code uses the history of chat prompts and responses as context for your current chat prompt. This means that you can ask follow-up questions or clarify your previous question without having to repeat the context. For example, you can ask "How does *this*
-differ from ...", "Now add a test case", "explain in more detail", and more.
+As you iterate and send multiple chat prompts in a chat session, VS Code uses the history of chat prompts and responses as context for your current chat prompt. This means that you can ask follow-up questions or clarify your previous question without having to repeat the context. For example, you can ask "How does *this* differ from ...", "Now add a test case", "explain in more detail", and more.
 
 At any time, you can create a new chat session by using the **New Chat** (`+`) button (`kb(workbench.action.chat.newChat)`) in the Chat view. This can be useful if you want to move to a different topic and avoid the previous context and history.
 
 To view the history of chat sessions, select the **Show Chats...** button in the Chat view or by using the **Chat: Show Chats...** command in the Command Palette. Select a history entry to open that chat session in the Chat view and continue the conversation.
 
 ![Screenshot of the Chat view with the Show Chats... button highlighted](images/copilot-chat/copilot-chat-view-show-chats.png)
 
-You can export all prompts and responses for a chat session in a JSON file with the **Chat: Export Chat...** command in the Command Palette.
+You can export all prompts and responses for a chat session in a JSON file with the **Chat: Export Chat...** command in the Command Palette. You can also copy a specific chat prompt or response to the clipboard in markdown format, by Ctrl + right-clicking the message and selecting "Copy", or copy the entire chat session by Ctrl + right-clicking the Chat view and selecting "Copy All".
```

Comment:
```suggestion
You can export all prompts and responses for a chat session in a JSON file with the **Chat: Export Chat...** command in the Command Palette. To copy a specific chat prompt or response to the clipboard in Markdown format, right-click the message and select **Copy**. To copy the entire chat session, right-click the Chat view and select **Copy All**.
```

Reply:
Oh, wow. I was pretty sure I had tried simple right-clicking, and only when that didn't work I sought out alternative ways to achieve that, ending up at https://github.com/orgs/community/discussions/160034. I guess I either misclicked when experimenting, or was instead looking for some icon in the UI. Thanks for the heads-up!